docs/Installatie.md

```diff
@@ -0,0 +1,146 @@
+# 1. Installatie
```

Comment:
Willen we de paragraaf nummering handhaven? Het is weliswaar makkelijker als je een keer wil verwijzen naar een specifieke paragraaf maar als we ergens een paragraaf tussenin willen plaatsen dan moeten we alle paragraafnummers daarna ook aan gaan passen.
Als we besluiten het wel te doen dan moeten we het ook overal doen. Hieronder hebben de laatste 4 paragrafen geen nummering.

Reply:
Goed punt. Maak ik een Issue van. Eerst zet ik alles 1 op 1 over. Daarna ga ik inhoudelijk voorstellen doen.